python/metatensor-operations/metatensor/operations/sort.py

```diff
@@ -4,6 +4,79 @@
 from ._classes import Labels, TensorBlock, TensorMap, torch_jit_is_scripting
 
 
+def _sort_single_gradient_block(
+    block: TensorBlock,
+    gradient_block: TensorBlock,
+    axes: List[str],
+    descending: bool,
+) -> TensorBlock:
+    """
+    Sorts a single TensorBlock without the user input checking and sorting of gradients
```

Comment:
```suggestion
    Sorts a single gradient TensorBlock. This is different from `_sort_single_block` because we need to update the sample differently (since gradient samples are pointers into the values samples)
```

Reply:
updated this doc and embedded suggested change into it